src/map/buyingstore.hpp

```diff
@@ -68,5 +68,6 @@ void do_init_buyingstore(void);
 
 void do_init_buyingstore_autotrade( void );
 void buyingstore_reopen( struct map_session_data* sd );
+void buyingstore_update_buyer_location(map_session_data &sd);
```

Comment:
```suggestion
void buyingstore_update(map_session_data &sd);
```
After your last commit it is not only location that is getting updated.